### PR TITLE
data Bishwajit-2810.json

### PR DESCRIPTION
**Pull request by Bishwajit-2810**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Bishwajit-2810.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "Open Source",
+    "C++",
+    "Java",
+    "Dart",
+    "Flutter",
+    "DevOps"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Bishwajit_2810" },
+    { "icon": "FaGithub", "url": "https://github.com/Bishwajit-2810" },
```
Comment (Viveksati5143) [inline data/Bishwajit-2810.json:14]:
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/Bishwajit-2810" }
```

Comment (Viveksati5143) [reply data/Bishwajit-2810.json:14]: I don't think `,` at the end is required

**Review by Viveksati5143 — CHANGES_REQUESTED** [review]

File: data/Bishwajit-2810.json (hunk 1 above)
Comment (Bishwajit-2810) [reply data/Bishwajit-2810.json]: Ok

**Comment by Bishwajit-2810** [comment]
I think that's Okay now kindly review it again please

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**